.babelrc

```diff
@@ -3,6 +3,7 @@
     "latest"
   ],
   "plugins": [
+    "transform-object-rest-spread",
```

Comment:
Odd, yeah it looks like it is seeing the reference, but not installing it properly?